.github/mergify.yml

```diff
@@ -1,6 +1,22 @@
 # Note: We do not use the rebase strategy to merge PRs, because that
 # loses information needed by changelog-d to associate commits with PRs.
 
+priority_rules:
+
+  - name: high priority
+    conditions:
+      - 'label=priority: high :fire:'
```

Comment:
I would have thought you would need to put the literal fire emoji here. Is this documented on Mergify's side?

Reply:
Mergify documents that it uses the GitHub API.

If you look at how the label is defined in our repo, it uses `:fire:` instead of the actual emoji, so that is what GitHub requires us to match on.

Reply:
Wonderful. :)